### PR TITLE
Adding the ocean_wave_tracing project

### PR DESCRIPTION
**Pull request by hevgyrt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/ocean_wave_tracing/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - ipython
+    - cmocean
+    - jupyterlab
+    - pytest
+    - simplejson
+
+test:
+  imports:
+    - simplejson
+    - simplejson.tests
```
Comment (carterbox) [inline recipes/ocean_wave_tracing/meta.yaml:44]: Test this package; not simplejson.

File: recipes/ocean_wave_tracing/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - xarray
+    - pyproj >=2.3
+    - cartopy
+    - pip
+    - yaml
+    - ipython
+    - cmocean
+    - jupyterlab
+    - pytest
+    - simplejson
```
Comment (carterbox) [inline recipes/ocean_wave_tracing/meta.yaml:39]: https://github.com/hevgyrt/ocean_wave_tracing/blob/v.1.0.3/pyproject.toml

These deps should match upstream.

File: recipes/ocean_wave_tracing/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  sha256: 54a06d09a0b57683686405ba15dbf51aed03a85ebd05a20321e2060bb1fae073
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (carterbox) [inline recipes/ocean_wave_tracing/meta.yaml:19]: Why do you need a c compiler for this package?

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/ocean_wave_tracing/meta.yaml (hunk 3 above)
Comment (hevgyrt) [reply recipes/ocean_wave_tracing/meta.yaml]: Thanks for pointing out. I don't. Removed

File: recipes/ocean_wave_tracing/meta.yaml (hunk 1 above)
Comment (hevgyrt) [reply recipes/ocean_wave_tracing/meta.yaml]: Thanks for pointing this out. Now removed

File: recipes/ocean_wave_tracing/meta.yaml (hunk 2 above)
Comment (hevgyrt) [reply recipes/ocean_wave_tracing/meta.yaml]: Thanks for pointing out. I have updated most of these. However, `yaml`, `ipython`, `pip`, and `jupyterlab` are still there. Unsure if that is needed for conda..

**Comment by hevgyrt** [comment]
@conda-forge-admin, please ping team

File: recipes/ocean_wave_tracing/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocean_wave_tracing-{{ version }}.tar.gz
+  sha256: 54a06d09a0b57683686405ba15dbf51aed03a85ebd05a20321e2060bb1fae073
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/ocean_wave_tracing/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python =>3.7
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:19]: The package metadata min python is 3.8.

```suggestion
    - python =>3.8
```

File: recipes/ocean_wave_tracing/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    - pip
+    - poetry
+  run:
+    - python >=3.7
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:23]:
```suggestion
    - python >=3.8
```

File: recipes/ocean_wave_tracing/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    - netcdf4 >=1.6
+    - xarray >=2022.6.0
+    - pyproj >=2.3
+    - cartopy >0.21
+    - cmocean >2.0
+    - pip
+    - yaml
+    - ipython
+    - jupyterlab
+    - pytest
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:37]: Are you sure you need those? They are not listed in the package metadata.

```suggestion
```

File: recipes/ocean_wave_tracing/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+test:
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:39]:
```suggestion
test:
  imports:
    - ocean_wave_tracing
```

File: recipes/ocean_wave_tracing/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - python =>3.8
+    - pip
+    - poetry
+  run:
+    - python >=3.8
+    - matplotlib-base
+    - matplotlib >=3.5
```
Comment (ocefpaf) [inline recipes/ocean_wave_tracing/meta.yaml:25]:
```suggestion
    - matplotlib-base >=3.5
```

**Pull request merged**